torch/csrc/jit/codegen/cuda/type.h

```diff
@@ -194,12 +194,12 @@ static constexpr std::array<ParallelType, 6> kParallelTypeThreads = {
     ParallelType::TIDy,
     ParallelType::TIDz};
 
-static constexpr std::array<ParallelType, 6> kParallelTypeBIDs = {
+static constexpr std::array<ParallelType, 3> kParallelTypeBIDs = {
     ParallelType::BIDx,
     ParallelType::BIDy,
     ParallelType::BIDz};
 
-static constexpr std::array<ParallelType, 6> kParallelTypeTIDs = {
+static constexpr std::array<ParallelType, 3> kParallelTypeTIDs = {
     ParallelType::BIDx,
     ParallelType::BIDy,
     ParallelType::BIDz};
```

Comment:
Can you also change `BID` to `TID`? Looks like this static variable isn't used yet, so nothing isn't impacted (fortunately).